documentation/Configuration.md

```diff
@@ -164,3 +164,25 @@ The following example shows a configuration file which requires developers to do
   }
 }
 ```
+
+## Naming Rules
+
+This section describes the features of naming rules which can be configured in **stylecop.json**. Each of the described properties are configured in the `namingRules` object, which is shown in the following sample file.
+
+```json
+{
+  "settings": {
+    "namingRules": {
+    }
+  }
+}
+```
+
+### Hungarian Notation
+
+The following properties are used to configure allowable Hungarian notation prefixes in StyleCop Analyzers.
+
+| Property | Default Value | Summary |
+| --- | --- | --- |
+| `allowCommonHungarianPrefixes` | **true** | Specifies whether a set of common prefixes ("as", "at", "by", "do", "go", "if", "in", "is", "it", "no", "of", "on", "or", "to") should be allowed |
+| `allowedHungarianPrefixes` | `[ ]` | Specifies allowable Hungarian notation prefixes |
```

Comment:
:thought_balloon: 

> Specifies additional prefixes which are allowed to be used in variable names. See the example below for more information.

Then right after the table you could have something like this:

> The following example shows a settings file which allows the common prefixes as well as the custom prefixes 'md' and 'cd'.
>
> ```json
> {
>   "settings": {
>     "namingRules": {
>       "allowedHungarianPrefixes": [
>         "cd",
>         "md"
>       ]
>     }
>   }
> }
> ```